File: cpp/09proc/Proc20.cpp
#include <iostream>
#include <cmath>
using namespace std;

double TriangleP(double a, double h);

int main()
{
	//Task("Proc20");
	double a, h;
	for (int i = 1; i <= 3; i++)
	{
		cin >> a >> h;
		cout << TriangleP(a, h);
	}
	
	return 0;
}

double TriangleP(double a, double h)
{
	double b = sqrt( pow(a/2, 2) + h*h );

	return 2*b + a;
}

File: php/09proc/proc20.php
<?php
function TriangleP($a, $h){
        $b = sqrt(pow($a / 2, 2) + $h * $h);
        $nat = $a + 2 * $b;
		$nat = number_format($nat, 2);
        return $nat;
}
$hisob = 1;
while($hisob <= 3){
	$a = mt_rand(1, 100) / 4;
	$h = mt_rand(1, 100) / 4;
	$natija = TriangleP($a, $h);
	$a = number_format($a, 2);
	$h = number_format($h, 2);
	echo "Асос: ".$a;
	echo "<br>Баландӣ: ".$h;
	echo "<br>Периметри секунҷа: ".$natija."<br><br>";
	$hisob++;
}
?>